models/experimental/functional_vit/tt/ttnn_functional_vit_highres.py

```diff
@@ -26,7 +25,7 @@ def vit_patch_embeddings(config, pixel_values, *, parameters, unittest_check=Fal
     stride_w = 1
 
     pixel_values = ttnn.reshape(pixel_values, (batch_size, img_h, img_w // patch_size, 4 * patch_size))
-    pixel_values = ttl.tensor.fold(pixel_values, stride_h, stride_w)
+    pixel_values = ttnn.experimental.tensor.fold(pixel_values, stride_h, stride_w)
```

Comment:
I think this op has changed https://github.com/tenstorrent/tt-metal/commit/7c67e3ec6de8bf9fbe1d0f218e6013db59bf9ea9